components/linkedin/actions/create-text-post-user/create-text-post-user.mjs

```diff
@@ -0,0 +1,51 @@
+import axios from "axios";
+
+export default {
+  key: "linkedin_create-text-post-user",
+  name: "Create a Text Post (User)",
```

Comment:
What is the "user" in this context? I see from the MSFT docs that the action is called "Creating a Share on LinkedIn" there. Can you create a post as an admin vs. a user?

Reply:
I see that the title in the linked issue is tied to that, gotcha. @dannyroosevelt we said the action should be called "Create a Text Post (User)" — maybe that's what Zapier had? — but the API docs call it "Creating a Share". What do you think the action title should be?

Reply:
I'm not sure where that came from. Zapier is "Create Share Update," so let's go with "Create a Share (consumer)". I think the "user" is referencing NOT posting to a company page.

Reply:
There will be 2 actions very similar:

1. Create a Text Post (User)
2. Create a Text Post (Organization)

What should be the correct key name?

Reply:
Create a Text Share (User): `linkedin_create-text-share-user`
Create a Text Share (Organization): `linkedin_create-text-share-organization`

Reply:
ok! this action has the first name